src/app/trainee/revalidation-history/revalidation-history.component.html

```diff
@@ -60,18 +56,26 @@
         </ng-container>
 
         <tr mat-header-row *matHeaderRowDef="columnsToDisplay"></tr>
-        <tr role="button" tabindex="0" mat-row *matRowDef="let element; columns: columnsToDisplay;" class="element-row"
+        <tr tabindex="0" mat-row *matRowDef="let element; columns: columnsToDisplay;" class="element-row"
             [class.expanded-row]="expandedElement === element"
             (click)="expandedElement = expandedElement === element ? null : element"
```

Comment:
what is `expandedElement` and is it possible to wrap `expandedElement === element ? null : element` in to a variable in the ts file or something as this code is used several times in the markup

Reply:
Honestly (head in hands) I have no clue and literally copied and pasted example on `https://material.angular.io/components/table/examples` search "expandable rows" on that page. Literally cut and paste. I will investigate though

Reply:
Tried some things and couldn't get it to play ball. :(

Reply:
ok no probs could you make a note of this and we can tackle it next sprint when we have more time